content/en/security/application_security/threats/security_signals.md

```diff
@@ -59,8 +59,9 @@ Use [Incident Management][4] to create an incident for a security signal.
 
 Use [Workflow Automation][5] to manually trigger a workflow for a security signal. 
 
+1. Make sure the workflow you want to run has a security trigger.
 1. On the [Signals Explorer][2] page, select a security signal.
-2. Scroll down to the **What is Workflow Automation** section. 
+2. Scroll down to the **What is Workflow Automation** section.
 3. Click **Run Workflow**.
 4. On the workflow modal, select the workflow you want to run. Depending on the workflow, you may be required to enter additional input parameters.
 5. Click **Run**.
```

Comment:
The numbers render correctly on the page, but just thought I'd point out that it's `1.` for the first two only.

Reply:
Oh good call, thanks!